src/UseCases/CancelDonation/CancelDonationUseCase.php

```diff
@@ -49,7 +58,7 @@ public function cancelDonation( CancelDonationRequest $cancellationRequest ): Ca
 
 		$this->sendConfirmationEmail( $donation );
```

Comment:
In the old application, there was additional error checking here so when the mail was not sent, the user got the message "There was an error sending you the confirmation mail, but please beleive use that we really really canceled your donation". 
Maybe add a TODO here?